tests/integration/construct_component_error_apply/testcomponent/index.ts

```diff
@@ -0,0 +1,47 @@
+import * as pulumi from "@pulumi/pulumi";
```

Comment:
The original file is missing the copyright.

```suggestion
// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.

import * as pulumi from "@pulumi/pulumi";
```